mobile/src/components/uam-groups-config/uam-configure-groups.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { FC, useEffect } from 'react';
+import { ScrollView, View } from 'react-native';
+
+import { AppScreenName, PaginationDefaultValue } from '~/common/enums/enums';
+import { GroupsUpdateRequestDto } from '~/common/types/types';
+import { Button, Input, Text } from '~/components/common/common';
+import {
+  PermissionsTable,
+  UsersTable,
+} from '~/components/uam-groups-config/components/components';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppNavigate,
+  useAppSelector,
+  usePagination,
+  useSelectedItems,
+} from '~/hooks/hooks';
+import { groupsCreationActions, uamGroupEditActions } from '~/store/actions';
+import { groupCreateClient } from '~/validation-schemas/validation-schemas';
+
+import { CREATE_GROUP_DEFAULT_PAYLOAD } from './common/constants/create-group-default.constants';
+import { styles } from './styles';
+
+const UAMConfigureGroup: FC = () => {
+  const navigation = useAppNavigate();
+  const dispatch = useAppDispatch();
+
+  const { group } = useAppSelector((state) => state.uamGroupEdit);
+
+  const { users, permissions } = useAppSelector(
+    (state) => state.uamGroupCreation,
+  );
+
+  const { page: usersPage, handlePageChange: handleUserPageChange } =
+    usePagination();
+
+  const {
+    page: permissionsPage,
+    handlePageChange: handlePermissionsPageChange,
+  } = usePagination();
+
+  const { control, handleSubmit, errors, reset, watch } =
+    useAppForm<GroupsUpdateRequestDto>({
+      defaultValues: CREATE_GROUP_DEFAULT_PAYLOAD,
+      validationSchema: groupCreateClient,
+    });
+
+  const {
+    items: permissionIds,
+    handleToggle: handleTogglePermissions,
+    setItems: setDefaultPermissionIds,
+  } = useSelectedItems<number>(group?.permissionIds ?? []);
+
+  const {
+    items: userIds,
+    handleToggle: handleToggleUsers,
+    setItems: setDefaultUserIds,
+  } = useSelectedItems<number>(group?.userIds ?? []);
+
+  const paginationForUsersTable = {
+    page: usersPage,
+    setPage: handleUserPageChange,
+  };
+
+  const paginationForPermissionsTable = {
+    page: permissionsPage,
+    setPage: handlePermissionsPageChange,
+  };
+
+  const groupPayload: GroupsUpdateRequestDto = {
+    name: watch.name,
+    permissionIds,
+    userIds,
+  };
+
+  const handleCreateOrEditGroup = async (): Promise<void> => {
```

Comment:
remove watch and get field value via payload
```suggestion
  const handleCreateOrEditGroup = async (payload: GroupsUpdateRequestDto): Promise<void> => {
```